nbs/docs/capabilities/08_cross_validation.ipynb

```diff
@@ -0,0 +1,1846 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "env: PYTORCH_ENABLE_MPS_FALLBACK=1\n"
+     ]
+    }
+   ],
+   "source": [
+    "#| hide\n",
+    "%set_env PYTORCH_ENABLE_MPS_FALLBACK=1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Cross-validation\n",
+    "\n",
+    ":::{.callout-warning collapse=\"true\"}\n",
+    "## Prerequesites\n",
+    "This Guide assumes basic familiarity with NeuralForecast. For a minimal example visit the [Quick Start](../getting-started/02_quickstart.ipynb)\n",
+    ":::\n",
+    "\n",
+    "To assess the performance of forecasting model, we must know how the model would have performed in the past. To do so, we must use *cross-validation*.\n",
```

Comment:
performance of a forecasting model, we can assess the model's performance on past data. This process is called *cross-validation*.